### PR TITLE
Adding SM cardfiles for the ggH H->tautau

### PR DESCRIPTION
**Pull request by maravin**
OK, try number 2. This time only the SM cardfiles have been requested to be merged. Once it is successful, I will repeat with the Pseudoscalar and max-mix ones. Thank you.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggHCPjets_v3/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggHCPjets_v3/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_proc_card.dat:12]: do you mind removing these default settings?

**Comment by agrohsje** [comment]
maybe you discussed before. what is the reason to have the full param_card instead of just the customize one ?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggHCPjets_v3/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_proc_card.dat (hunk 1 above)
Comment (maravin) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggHCPjets_v3/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8/GluGluToHToTauTauPlusTwoJets_M125_13TeV_amcatnloFXFX_pythia8_proc_card.dat]: Done for all *proc_card.dat files in four processes. Pushed to git.

**Comment by maravin** [comment]
I was told by the author of the card files that there was no reason in particular. However, we do need the MC samples produced, and these datacards have been tested. So I propose we proceed with what we have, rather than spending valuable time on redoing the work that amounts to the same result?

**Comment by agrohsje** [comment]
i cannot spot the ufo here: https://cms-project-generators.web.cern.ch/cms-project-generators/
do you mind putting it in a public directory so i can copy? thanks!
ok for the param_card. the customize_card is always easier to review but if you say it was tested, no problem.

**Comment by agrohsje** [comment]
@sihyunjeon : besides the ufo fine for me to merge.

**Comment by sihyunjeon** [comment]
Looks fine, as Alexander said send us the model files

**Pull request merged**